grpc-api/src/main/java/io/deephaven/grpc_api/util/GrpcUtil.java

```diff
@@ -22,7 +21,12 @@ public static void rpcWrapper(final Logger log, final StreamObserver<?> response
         try (final SafeCloseable ignored = LivenessScopeStack.open()) {
             lambda.run();
         } catch (final StatusRuntimeException err) {
-            log.error().append(err).endl();
+            if (err.getStatus().equals(Status.UNAUTHENTICATED)) {
+                // most of the time we do not want to log this
+                log.debug().append("ignoring unauthenticated request").endl();
```

Comment:
Should we log (to debug) what we're ignoring?
"Ignoring unauthenticated request"